addons/mail/static/src/models/rtc.js

```diff
@@ -754,11 +755,10 @@ registerModel({
             if (rtcSession) {
                 rtcSession.reset();
             }
-            const dataChannel = this._dataChannels[sessionId];
-            if (dataChannel) {
-                dataChannel.close();
+            const rtcDataChannel = rtcSession.rtcDataChannel;
```

Comment:
`rtcSession` can already be deleted in some asynchonous calls to `_removePeer()`, that's why it's guarded above, maybe this part should also be in the guard to avoid tracebacks when trying to read `rtcDataChannel` of `undefined`.

Reply:
`delete()` is on `RtcDataChannel`: it changes the modelling so that there is no more `RtcDataChannel` related to this `RtcSession`. `close()` is on `RtcDataChannel/dataChannel`, and it's auto-closed on deletion of the `RtcDataChannel` (see `RtcDataChannel/_willDelete`).